mu-plugins/websub-compat.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Plugin Name: WebSub Backwards Compatibility
+ * Version: 1.0.0
+ * Author: WordPress.com Special Projects
+ * Author URI: https://wpspecialprojects.wordpress.com/
```

Comment:
Could we add a plugin description so we have easy visibility of what this does from the Plugins view?